src/js/pyodide.ts

```diff
@@ -65,6 +65,33 @@ function wrapPythonGlobals(globals_dict: PyDict, builtins_dict: PyDict) {
   });
 }
 
+/**
+ * This function is called right after the Python interpreter is initialized,
+ * but before any Python code is run. It sets up some unfinished parts of the
+ * Python environment.
+ * @private
+ */
+function postInitializePython(Module: any) {
+  const code = `
+import sys, _imp, encodings, codecs
+codecs.unregister(encodings.search_function)
+del sys.modules["encodings"]
+del sys.modules["encodings.utf_8"]
+del sys.modules["encodings.aliases"]
+_imp._override_frozen_modules_for_tests(-1)
```

Comment:
Shouldn't you set this back to 0 afterwards?

Reply:
I think it is not necessary. All frozen modules are already imported and saved in `sys.modules`, so we won't get extra performance advantages from setting the value back.